R/R.r

```diff
@@ -1,6 +1,6 @@
 # R("-e 'str(as.list(Sys.getenv()))' --slave")
 R <- function(options, path = tempdir(), env_vars = NULL, ...) {
-  options <- paste("--vanilla", options)
+  options <- paste("--no-site-file", "--no-environ", options)
```

Comment:
Not at all. I was relying on `?Startup` which reads

> The command-line option --vanilla implies --no-site-file, --no-init-file, --no-environ and (except for R CMD) --no-restore

but this refers to startup only and does not seem to list what *else* is implied by --vanilla. Thanks.